### ISSUE TITLE
zephir make error 

### ISSUE THREAD

**Issue by ghost** [zephir-lang/zephir#1273]
I can`t make

/usr/local/zephir/runtime/builder.c:921:7: warning: assignment makes pointer from integer without a cast [enabled by default]
  cast = LLVMBuildZExt(context->builder, type, LLVMInt32Type(), ""); // zext i8 %4 to i32
       ^
/usr/local/zephir/runtime/builder.c:922:46: error: ‘LLVMIntEQ’ undeclared (first use in this function)
  condition = LLVMBuildICmp(context->builder, LLVMIntEQ, // icmp eq i32 %5, 1
                                              ^
/usr/local/zephir/runtime/builder.c:928:22: warning: initialization makes pointer from integer without a cast [enabled by default]
  LLVMValueRef func = LLVMGetBasicBlockParent(LLVMGetInsertBlock(context->builder));
                      ^
/usr/local/zephir/runtime/builder.c:930:33: warning: initialization makes pointer from integer without a cast [enabled by default]
  LLVMBasicBlockRef then_block = LLVMAppendBasicBlock(func, "then");
                                 ^
/usr/local/zephir/runtime/builder.c:931:33: warning: initialization makes pointer from integer without a cast [enabled by default]
  LLVMBasicBlockRef else_block = LLVMAppendBasicBlock(func, "else");
                                 ^
/usr/local/zephir/runtime/builder.c:932:34: warning: initialization makes pointer from integer without a cast [enabled by default]
  LLVMBasicBlockRef merge_block = LLVMAppendBasicBlock(func, "merge-if");
                                  ^
/usr/local/zephir/runtime/builder.c:940:6: warning: assignment makes pointer from integer without a cast [enabled by default]
  ptr = LLVMBuildLoad(context->builder, symbol_ref, ""); // load %struct._zval_struct*\* %1, align 8
      ^
/usr/local/zephir/runtime/builder.c:941:13: warning: assignment makes pointer from integer without a cast [enabled by default]
  indices[0] = LLVMConstInt(LLVMInt32Type(), 0, 0);
             ^
/usr/local/zephir/runtime/builder.c:942:13: warning: assignment makes pointer from integer without a cast [enabled by default]
  indices[1] = LLVMConstInt(LLVMInt32Type(), 0, 0);
             ^
/usr/local/zephir/runtime/builder.c:943:6: warning: assignment makes pointer from integer without a cast [enabled by default]
  ref = LLVMBuildInBoundsGEP(context->builder, ptr, indices, 2, ""); // getelementptr inbounds %struct._zval_struct\* %6, i32 0, i32 2
      ^
/usr/local/zephir/runtime/builder.c:944:10: warning: assignment makes pointer from integer without a cast [enabled by default]
  bitcast = LLVMBuildBitCast(context->builder, ref, LLVMPointerType(LLVMDoubleType(), 0), ""); //bitcast %union._zvalue_value\* %7 to double*
          ^
/usr/local/zephir/runtime/builder.c:945:13: warning: assignment makes pointer from integer without a cast [enabled by default]
  then_value = LLVMBuildLoad(context->builder, bitcast, ""); // load i64\* %10, align 8
             ^
/usr/local/zephir/runtime/builder.c:948:13: warning: assignment makes pointer from integer without a cast [enabled by default]
  then_block = LLVMGetInsertBlock(context->builder);
             ^
/usr/local/zephir/runtime/builder.c:956:13: warning: assignment makes pointer from integer without a cast [enabled by default]
  else_block = LLVMGetInsertBlock(context->builder);
             ^
/usr/local/zephir/runtime/builder.c:962:6: warning: assignment makes pointer from integer without a cast [enabled by default]
  phi = LLVMBuildPhi(context->builder, LLVMDoubleType(), ""); //phi i64 [ %11, %7 ], [ %14, %12 ]
      ^
make: **\* [builder.lo] Error 1

**Comment by ghost** [comment]
Who can help me

**Comment by steffengy** [comment]
Duplicate #1204 .
TLDR: That code is very experimental and wasn't touched a very long time, so there's no support for it at the time aswell as no focus on it.

**Comment by ghost** [comment]
Oh....

thanks
